src/main/java/ch/uzh/ifi/access/course/util/RepoCacher.java

```diff
@@ -124,6 +124,7 @@ private void cacheRepo(File file, Object context) {
                 a.addExercise(exercise);
                 next_context = exercise;
             } else if (file.getName().startsWith(PUBLIC_FOLDER_NAME)) {
+                // TODO investigate if it would make more sense to leave the root folder in tha path
```

Comment:
👍 
But let's just commit it as it is, and we can figure this out later

Reply:
Jup. Just marked it so we don't forget about it